deployments/gcp-uscentral1b/config/common.yaml

```
@@ -20,6 +20,10 @@ pangeo:
           - name: home
             mountPath: /home/jovyan
             subPath: "uscentral1b/{username}"
+      extraEnv:
+        # Scratch storage within the cluster
+        PANGEO_SCRATCH: "gcs://pangeo-scratch"
```

To be consistent with https://github.com/pangeo-data/pangeo-docker-images/pull/89, this needs to be different
```suggestion
        PANGEO_SCRATCH_PREFIX: "gcs://pangeo-scratch"
```

---

Whoops, thanks.